yjit/src/codegen.rs

```diff
@@ -3118,6 +3118,21 @@ fn c_method_tracing_currently_enabled(jit: &JITState) -> bool
     unsafe { rb_c_method_tracing_currently_enabled(jit.ec.unwrap() as *mut rb_execution_context_struct) }
 }
 
+// Called at runtime to build hashes of passed kwargs
+// Similar to args_kw_argv_to_hash
```

Comment:
Would be good to comment here on what the hash maps
keywords (symbols?) to VALUEs ?